docs/trace/get-started.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Get Started
+parent: Trace Analytics
+nav_order: 1
+---
+
+# Get started with Trace Analytics
+
+Open Distro for Elasticsearch Trace Analytics consists of two components---Data Prepper and the Trace Analytics Kibana plugin---that fit into the OpenTelemetry and Elasticsearch ecosystems. The Data Prepper repository has several [sample applications](https://github.com/opendistro-for-elasticsearch/Data-Prepper/tree/master/examples) to help you get started.
+
+
+## Basic flow of data
+
+![Data flow diagram from a distributed application to Open Distro for Elasticsearch](../../images/ta.svg)
+
+1. Trace Analytics relies on you adding instrumentation to your application and generating trace data. The [OpenTelemetry documentation](https://opentelemetry.io/docs/) contains example applications for many programming languages that can help you get started, including Java, Python, Go, and JavaScript.
+
+1. The [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/getting-started/) receives data from the application and formats it into OpenTelemetry data.
+
+1. [Data Prepper](../data-prepper/) processes the OpenTelemetry data, transforms it for use in Open Distro for Elasticsearch, and indexes it on an Elasticsearch cluster.
+
+1. The [Trace Analytics Kibana plugin](../ta-kibana/) displays the data in near real-time as a series of charts and tables, with an emphasis on service architecture, latency, error rate, and throughput.
+
+
+## Jaeger HotROD
```

Comment:
I am okay with this, but Rajiv is using the sample we wrote in his launch blog. Just check with him.

Reply:
Also, The Jaeger Hotrod setup is different from the `images/ta.svg`. In the Jaeger hotrod the flow is,

Application -> Jaeger-Agent -> Opentelemetry collector -> ODFE -> ODFE Kibana.